### PR TITLE
Timeline component

### PR DESCRIPTION
**Pull request by j8seangel**
Moving time bar component to this repo https://github.com/GlobalFishingWatch/map-components/issues/1

image

- Created a sandbox page to test the time bar.
- Added a simple router to navigate between components.

Nice to think about:
- Rollup warning with https://github.com/d3/d3-selection/issues/168#issuecomment-401059437
- Dependencies susceptibles to be `peerDependencies`.

### CODE REVIEW

File: sandbox/package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@globalfishingwatch/map-components": "link:..",
```
Comment (nerik) [inline sandbox/package.json]: 🤯

File: src/index.js [hunk 2]
```diff
@@ -1 +1,5 @@
 export { default as MiniGlobe } from './miniglobe/miniglobe'
+export { default as Timebar } from './timebar/Timebar'
+export { default as TimebarActivity } from './timebar/charts/Activity'
```
Comment (nerik) [inline src/index.js]: Not 100% sure about that. I'd rather being able to do
```
import Timebar from '@globalfishingwatch/map-components'
import { VesselEvents } from '@globalfishingwatch/map-components/timebarCharts'
```
than
```
import {
  Timebar,
  VesselEvents
} from '@globalfishingwatch/map-components'
```

**Comment by nerik** [comment]
Any idea how to fix this?
screenshot 2019-02-05 at 17 43 09

**Comment by j8seangel** [comment]
Added components description

**Pull request merged**